src/main/java/gov/loc/mets/HelperTest.java

```diff
@@ -20,16 +18,17 @@
  * You should have received a copy of the GNU Lesser General Public License
  * along with this library. If not, see <http://www.gnu.org/licenses/>.
  ******************************************************************************/
-
-import gov.loc.mets.MetsType.MetsHdr;
-import gov.loc.mets.MetsType.MetsHdr.Agent;
+package gov.loc.mets;
 
 import java.io.File;
 import java.io.IOException;
 import java.util.HashMap;
 
 import org.apache.xmlbeans.XmlOptions;
 
+import gov.loc.mets.MetsType.MetsHdr;
```

Comment:
Same again: moving import statements from alphabetical ordering to mixed ordering.